addons/skin.estuary/xml/MusicVisualisation.xml

```diff
@@ -41,62 +41,107 @@
 			</control>
 		</control>
 		<control type="group">
+			<visible>String.IsEmpty(Player.Art(fanart)) | [!Visualisation.Enabled + Skin.HasSetting(hide_background_fanart)] | [Visualisation.Enabled + String.IsEmpty(Window(Visualisation).Property(ArtistSlideshow.Image))]</visible>
```

Comment:
same..don't get the visualisation/artistslideshow dependency.
i'm also unsure if we should take the hide_background_fanart skin setting into account.

this is a tough one to get it right

the condition of the animation below also might need to be updated once we get this visible condition right.
(ie. no point in animation something that isn't visible)